tests/util/test_yaml_extension.cpp

```diff
@@ -131,3 +131,57 @@ TEST(YamlEigenExtension, TsrTransforms)
+        "    \"offset\": [                                             \n"
+        "      [                                                       \n"
+        "        -0.0068393994632501,                                  \n"
+        "        -0.09054787493933,                                    \n"
+        "        -0.99586861832219,                                    \n"
+        "        0.14460904118031                                      \n"
+        "        ]                                                     \n"
+        "      ,                                                       \n"
+        "      [                                                       \n"
+        "        -0.9999766064057,                                     \n"
+        "        0.00052349074982687,                                  \n"
+        "        0.0068200145734176,                                   \n"
+        "        -0.00097599685803373                                  \n"
+        "        ]                                                     \n"
+        "      ,                                                       \n"
+        "      [                                                       \n"
+        "        -9.6209816943679e-5,                                  \n"
+        "        0.99589196617977,                                     \n"
+        "        -0.090549337061432,                                   \n"
+        "        -0.033644917605727                                    \n"
+        "      ]                                                       \n"
+        "      ,                                                       \n"
+        "      [                                                       \n"
+        "        0,                                                    \n"
+        "        0,                                                    \n"
+        "        0,                                                    \n"
+        "        1                                                     \n"
+        "      ]                                                       \n"
+        "    ]                                                         \n"
+        "  }                                                           \n"
+        "}                                                             \n";
+
+  auto root = YAML::Load(jsonString);
+  auto tag124 = root["tag124"]["offset"].as<Eigen::Isometry3d>();
+
+  Eigen::Isometry3d expectedTag124 = Eigen::Isometry3d::Identity();
+  expectedTag124.linear()
+      << -0.0068393994632501, -0.09054787493933   , -0.99586861832219  ,
+         -0.9999766064057   ,  0.00052349074982687,  0.0068200145734176,
+         -9.6209816943679e-5,  0.99589196617977   , -0.090549337061432 ;
+  expectedTag124.translation()
+      << 0.14460904118031, -0.00097599685803373, -0.033644917605727;
+  EXPECT_TRUE(tag124.matrix().isApprox(expectedTag124.matrix()));
+}
```

Comment:
Should we also test that we get representation exceptions with invalid YAML?  Like mismatched sizes?